scripts/scan-txlog/unclaimed_sv_rewards.py

```diff
@@ -651,21 +651,18 @@ def handle_sv_reward_coupon_exercise(self, transaction, event, status):
         self.process_events(transaction, event.child_event_ids)
 
     def _calculate_amount(self, reward, mining_round_info):
-        return (
-            self.weight * mining_round_info.issuance_per_sv_reward
-            if self._is_weight_ok(reward)
-            else 0
-        )
+        return self._calculate_weight(reward) * mining_round_info.issuance_per_sv_reward
 
-    def _is_weight_ok(self, reward):
+    def _calculate_weight(self, reward):
         available_weight = reward.weight - self.already_minted_weight
```

Comment:
Can you guarantee that this never goes negative? I think that's what I meant with clamp to zero earlier.